### PR TITLE
Reduced code smells

### PR DESCRIPTION
**Pull request by balaktsisc**
Used Pylint and PyCharm code inspection to detect and remove or refactor some of the code smells of the project, without affecting its functionality.

### CODE REVIEW

File: src/commands/level.py [hunk 1]
```diff
@@ -67,10 +70,11 @@ async def rank(message: discord.Message, parameters: str, client: discord.Client
     "category": Category.LEVELING,
     "description": "Shows a list of all users and their XP"
 })
-async def leaderboards(message: discord.Message, parameters: str, client: discord.Client, first_execution=True, op=None, page_cache=0) -> None:
+async def leaderboards(message: discord.Message, parameters: str,
```
Comment (Neron-png) [inline src/commands/level.py:73]: Breaking

File: src/commands/misc.py [hunk 2]
```diff
@@ -26,51 +29,53 @@ async def pad(message: discord.Message, parameters: str, client: discord.Client)
     """Spaces out your text"""
     if parameters == "":
         raise CommandSyntaxError("You must specify text to space out.")
-    elif len(parameters) > 1000:
+    if len(parameters) > 1000:
```
Comment (Neron-png) [inline src/commands/misc.py:32]: Bad practice, less intuitive

File: src/commands/misc.py [hunk 3]
```diff
@@ -96,18 +101,22 @@ async def aa(message: discord.Message, parameters: str, client: discord.Client)
...
-    last_was_lower_m = False  # Was the last character converted to a lowercase m? Used to make "HmmHmm" instead of "HmmMmm"
-    sentence = "" # Output string
+    last_was_alpha = False  # Was the last character part
+    # of the alphabet? Used to detect the start of a word
+    last_was_h = False  # Was the last character converted
+    # to an H or h? Used to prevent 'H's without 'm's
+    last_was_lower_m = False  # Was the last character
+    # converted to a lowercase m? Used to make "HmmHmm"
+    # instead of "HmmMmm"
+    sentence = ""  # Output string
```
Comment (Neron-png) [inline src/commands/misc.py:119]: This is just noise

File: src/commands/misc.py [hunk 4]
```diff
@@ -148,71 +157,74 @@ async def villager(message: discord.Message, parameters: str, client: discord.Cl
...
 
 
 @command({
     "syntax": "faderolecolour <role> <colour> <duration>",
     "aliases": ["fadecolour"],
     "role_requirements": {configuration.MODERATOR_ROLE},
     "category": Category.OTHER,
-    "description": "Fades the colour of a role to an other colour (hex) in a certain time period (h, d or m)"
+    "description": "Fades the colour of a role to an other \
+                    colour (hex) in a certain time period (h, d or m)"
```
Comment (Neron-png) [inline src/commands/misc.py:170]: Unnecessary, more noise

**Comment by Bobby-McBobface** [comment]
Mostly looks good to me, except for that breaking change in the leaderboard function.
Also go find other OSS projects to work on, why do this one >:(

**Pull request closed without merging**